src/language-service/src/schemas/types.ts

```diff
@@ -60,6 +61,12 @@ export type DeviceClassesBinarySensor =
   | "vibration"
   | "window";
 
+/**
+ * From: https://github.com/home-assistant/core/blob/dev/homeassistant/components/camera/__init__.py
+ */
+export type DeviceClassesCamera =
+  | "camera"
```

Comment:
Cameras do not have device classes

Reply:
Hi @frenck and thanks for your feedback. 

I was trying to resolve the following issue when using a device selector in a blueprint.

![image](https://user-images.githubusercontent.com/24822223/207260969-4ae96dc4-3e73-451e-9558-10ccf803eadd.png)

In this instance, the frigate integration creates entities which have a device_class of camera and filtering the selector using it works, but has the error in vscode

Reply:
That is invalid code. There is no “camera” device class in Home Assistant.

Reply:
It works, because frigate creates entities with that device class. and the selecter is correctly filtered based on that code. But if I am understanding correctly, instead the integration should be changed to not create entities with device_class: camera ?

![image](https://user-images.githubusercontent.com/24822223/207276371-54466959-b7b6-4986-8986-9eefaef5b6f5.png)

Reply:
> It works, because frigate creates entities with that device class. 

That is their problem. Please report it with Frigate, which is a third-party integration. There is no `camera` device class in Home Assistant. Setting a device class on a camera entity is invalid.